src/fragments/lib-v1/datastore/native_common/setup-auth-rules.mdx

```diff
@@ -55,7 +55,7 @@ The following are commonly used patterns for static group authorization. For mor
 
 ```graphql
 type YourModel @model @auth(rules: [{ allow: groups,
-                                      groups: ["Admin"] }]) {
+  groups: ["Admin"] }]) {
```

Comment:
Similar to other comments, formatting change seems like it is probably an improvement but out of scope for this PR. Lets remove changes that are not related to v0/v1 unless they are significant improvements (ex: fixing an obvious issue)